apps/docs/src/.vitepress/components/OnyxTypography.vue

```diff
@@ -49,7 +55,16 @@ const currentTab = ref<AvailableFontTab>(AVAILABLE_FONT_TABS[0]);
         :wide-name="props.wideName"
       >
         <template #name>
+          <OnyxHeadline
+            v-if="HEADLINE_TYPES.includes(token.htmlTag as HeadlineType)"
```

Comment:
Don't we want to prevent that something not part of HEADLINE_TYPES is passed? / Do we even want to support something else?

1. update type in props
<img width="1024" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/2cbc5889-9241-4bf1-ae68-551eb02a2ff2">

2. remove v-if

```suggestion
```
3. remove v-else block
<img width="890" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/bccae4c4-63e6-4d85-aceb-bb8527c6bd00">

Reply:
lint all + build all scripts would be happy with that adjustment :)

Reply:
We also need `<p>` and `<a>` :) But I simplified it quiet a lot, can you please take another look?

Reply:
NICE! 😊